background/get-addon-settings.js

```diff
@@ -46,6 +46,15 @@ chrome.storage.sync.get(["addonSettings", "addonsEnabled"], ({ addonSettings = {
         addonsEnabled["block-count"] = true;
       }
       if (addonId === "editor-dark-mode") {
+        if (settings.textShadow && addonsEnabled["custom-block-text"] === undefined) {
+          // Transition v1.23 to v1.24
+          // Moved text shadow option to the custom-block-text addon
+          madeChangesToAddon = madeAnyChanges = true;
+          delete settings.textShadow;
+          addonsEnabled["custom-block-text"] = true;
```

Comment:
This new addon should only be enabled if dark-www was enabled at the time of the transition.

Reply:
You mean `editor-theme-3` or `editor-dark-mode`?

Reply:
I meant editor dark mode, whoops